controller/ingress_entry.go

```diff
@@ -64,6 +66,22 @@ func (e IngressEntry) validate() error {
 	if e.ServicePort == 0 {
 		return errors.New("missing service port")
 	}
+
+	var invalidAllowEntries []string
+	for _, allowEntry := range e.Allow {
+		if net.ParseIP(allowEntry) == nil {
+			if _, _, err := net.ParseCIDR(allowEntry); err != nil {
+				invalidAllowEntries = append(invalidAllowEntries, allowEntry)
+			}
+		}
+	}
+
+	if len(invalidAllowEntries) == 1 {
+		return fmt.Errorf("invalid entry in sky.uk/allow: %s", invalidAllowEntries[0])
+	} else if len(invalidAllowEntries) > 1 {
+		return fmt.Errorf("invalid entries in sky.uk/allow: %s", strings.Join(invalidAllowEntries, ","))
```

Comment:
I think we should indicate which host the entry in invalid for  as there could be a large number of ingresses.

Reply:
besides in when the entry is empty, it would be quite hard to find which one is missing

Reply:
Actually, I ought to add a test that covers the case of empty entries as well.